jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/IoUtilTest.java

```diff
@@ -16,38 +16,37 @@
 import java.io.IOException;
 import java.net.ServerSocket;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
-public class IoUtilTest {
+@SuppressWarnings("unused")
+class IoUtilTest {
 
     @Test
-    public void findOpenPort() throws IOException {
+    void findOpenPort() throws IOException {
```

Comment:
@rohanKanojia  this is pointed out by sonar as a code smell, so do we need to assert this or ignore it?

Reply:
We can add an assertion here to verify created `ss` object is not null